### DISCUSSION TITLE
Could you theoretically compile kade engine to JVM?

### DISCUSSION THREAD

**Discussion by skypentnoob** [KadeArchive/Kade-Engine#2085] [Q&A]
I don't know about Haxe really my only previous experience is JavaScript, and I'm trying to learn Java. Theoretically, could you use JVM for Kade Engine by compiling the code to that instead of something like, say, an exe file, or HashLink, or neko?

**Comment by isakube** [comment]
JVM? more like JV HMMMMMMMMMMMMMMMMM

maybe, im an idiot with literally any other language

**Comment by skypentnoob** [comment]
JVM is a haxe compiling target. If you somehow don't know JVM is the Java Virtual Machine.
i think you would have to edit a lot of code for Kade Engine in JVM to work. Although nobody has concretely answered this question yet

**Reply by isakube** [reply]
> JVM is a haxe compiling target.

I am also an idiot with Haxe

With enough modification it could work. I mean, ajdhfalksdfgasdhjfgadsjlfgdafsljdjghlhgfdsghjfsadjghdfasgljfasjgfhsagjhfdfgdajfdah%&*&*%^%^&*%&*^%^&*gfkhjaskdgkjhDSGFJhksgfjhgsrugf89&D6789D879dFS6789DFS869df7gauoyifger2giksufvtfgiuygriufgewc7bcrw89cqw7g

> If you somehow don't know JVM is the Java Virtual Machine.

somebody's passive-aggressive in an unintentional manner
